src/libData/AccountData/MBnForwardedTxnEntry.h

```diff
@@ -17,27 +17,28 @@
  * program files.
  */
 
-#ifndef __FORWARDEDTXNENTRY_H__
-#define __FORWARDEDTXNENTRY_H__
+#ifndef __MBNFORWARDEDTXNENTRY_H__
+#define __MBNFORWARDEDTXNENTRY_H__
 
 #include "AccountStore.h"
 #include "Transaction.h"
 #include "TransactionReceipt.h"
-#include "libData/BlockData/BlockHeader/BlockHashSet.h"
+#include "libData/BlockData/Block/MicroBlock.h"
 
-struct ForwardedTxnEntry {
-  uint64_t m_blockNum;
-  BlockHash m_hash;
+struct MBnForwardedTxnEntry {
+  MicroBlock m_microBlock;
   std::vector<TransactionWithReceipt> m_transactions;
 
-  friend std::ostream& operator<<(std::ostream& os, const ForwardedTxnEntry& t);
+  friend std::ostream& operator<<(std::ostream& os,
+                                  const MBnForwardedTxnEntry& t);
 };
 
-inline std::ostream& operator<<(std::ostream& os, const ForwardedTxnEntry& t) {
-  os << "<ForwardedTxnEntry>" << std::endl
-     << "m_blockNum : " << t.m_blockNum << std::endl
-     << "m_hash : " << t.m_hash.hex();
+inline std::ostream& operator<<(std::ostream& os,
```

Comment:
can include mb hash?

Reply:
I wish someday we can have a printing operator for all the block data, then no need to manually log these every time, will put an issue for that